spec/result.json

```diff
@@ -0,0 +1,156 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://www.w3.org/2023/sparql/results.json",
+    "anyOf": [
+        {
+            "type": "object",
+            "required": [
+                "head",
+                "boolean"
+            ],
+            "properties": {
+                "head": {
+                    "type": "object",
+                    "properties": {
+                        "link": {
+                            "type": "array",
+                            "items": {
+                                "type": "string",
+                                "format": "iri"
+                            }
+                        }
+                    },
+                    "additionalProperties": false
```

Comment:
Do we really want to disallow additional properties here and later?
This explicitly forbids extension points, which is not forbidden in the current spec AFAICS.

Reply:
Great question!

Reply:
> RDF terms are not extensible

I would even allow those to be extensible (since that is not explicitly disallowed in the spec at the moment).

My suggestion would be to remove all occurences of `additionalProperties` throughout the whole schema.